demisto_sdk/commands/common/tests/release_notes_test.py

```diff
@@ -393,3 +394,78 @@ def test_has_release_notes_been_filled_out(release_notes, filled_expected_result
+def test_is_docker_image_same_as_yml(release_notes_content, yaml_content, filled_expected_result, pack: Pack):
+    """
+    Given
+    - Case 1: Validate for integrations, version match.
+    - Case 2: Validate for scripts, version match.
+    - Case 3: Valid when version doesnt match.
+    - Case 4: Release notes appear but docker not changed.
+
+    When
+    - Running validation on release notes.
+
+    Then
+    - Ensure validation correctly identifies valid release notes.
+    - Case 1: Should print nothing and return True.
+    - Case 2: Should print nothing and return True.
+    - Case 3: Should return the prompt "Release notes dockerimage version does not match yml dockerimage version." and
+            return False.
+    - Case 4: Should print nothing and return True.
+    """
+    rn = pack.create_release_notes(version='1_0_1', content=release_notes_content)
+    integration1 = pack.create_integration()
+    integration1.yml.update(yaml_content)
+    validator = ReleaseNotesValidator(rn.path, modified_files=[integration1.yml.path],
+                                      pack_name=os.path.basename(pack.path))
+    assert validator.is_docker_image_same_as_yml() == filled_expected_result
+
+
+TEST_RELEASE_NOTES_TEST_BANK_4 = [
+    ("\n#### Integrations\n##### Integration name\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.\n" +
+     "#### Scripts\n##### Script name\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.",
+     {'Integrations': '##### Integration name\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.',
+      'Scripts': '##### Script name\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.'}, "\n#### "),
+    ("\n#### Integrations\n##### Integration name1\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.\n" +
+     "##### Integration name2\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.",
+     {'Integrations': "##### Integration name1\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.\n" +
+      "##### Integration name2\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*."}, "\n#### "),
+    ("##### Integration name1\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.\n" +
+     "##### Integration name2\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.",
+     {'Integration name1': '- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.\n',
+      'Integration name2': '- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.'}, "##### "), ]
+
+
+@pytest.mark.parametrize('release_notes_content, filled_expected_result, splitter', TEST_RELEASE_NOTES_TEST_BANK_4)
+def test_get_categories_from_rn(release_notes_content, filled_expected_result, splitter, pack: Pack):
+    """
+    Given
+    - Case 1: Validate for 1 integration and one script
+    - Case 2: Validate for integration with 2 different integrations
+    - Case 3: Validate for splitter param
```

Comment:
After we separate the get_categories_from_rn  function into 2 let's separate the test also.
```suggestion
    - Case 1: RN containing an integration with a docker image update and a script with a docker image update.
    - Case 2: RN containing 2 different integrations with a docker image update.
    - Case 3: Validate for splitter param
```